package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "ad-ldap-connector",
-  "version": "6.1.8",
+  "version": "6.1.9",
```

Comment:
If we are forcing Node18, this should be a major version bump imo. I'd personally like to avoid that if possible for ease of customer updating.

Seems like this would at least be a minor version bump as support for a new runtime is being added.